src/config/app.py

```diff
@@ -8,7 +8,7 @@
 def create_app(environment_name: str):
     # Init flask openapi3
```

Comment:
@PedroTeixeira-dev If you want you can remove these comments 😉

Reply:
done